operate/services/manage.py

```diff
@@ -853,6 +866,38 @@ async def funding_job(
                     )
                 await asyncio.sleep(60)
 
+    async def healthcheck_job(
+        self,
+        hash: str,
+        loop: t.Optional[asyncio.AbstractEventLoop] = None,
+    ) -> None:
+        """Start a background funding job."""
+        loop = loop or asyncio.get_event_loop()
+        failed_health_checks = 0
+
+        with ThreadPoolExecutor() as executor:
+            while True:
+                try:
+                    # Check the service health
+                    healthy = await loop.run_in_executor(
+                        executor,
+                        check_service_health,
```

Comment:
`check_service_health` is already non-blocking no need to use the thread pool executor here, just

```python
healthy = await check_service_health()
```

Reply:
Addressed